projects/aca-content/src/lib/components/search/search-input-control/search-input-control.component.ts

```diff
@@ -76,6 +79,7 @@ export class SearchInputControlComponent implements OnDestroy {
   clear() {
     this.searchTerm = '';
     this.searchChange.emit('');
+    this.location.back();
```

Comment:
why it should go back in history after clearing search input?

Reply:
this is added to navigate the user to the previous route when the user clicks on the cross (clear) icon.

Reply:
but what if user wants to clear the input and type another one? have you confirmed this behaviour with the UX team?

Reply:
This behavior was added as per this animation prototype when the user clicks on the search icon in the header. https://www.figma.com/proto/KhXtphgH38kcqa8tJTrRmS/ADW-implementation-Q%26A?node-id=2%3A1967&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=2%3A1967